[2023-06-01T11:20:08.934+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: dag_with_postgres_operator_v03.delete_data_from_table scheduled__2022-01-19T00:00:00+00:00 [queued]>
[2023-06-01T11:20:09.012+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: dag_with_postgres_operator_v03.delete_data_from_table scheduled__2022-01-19T00:00:00+00:00 [queued]>
[2023-06-01T11:20:09.014+0000] {taskinstance.py:1283} INFO - 
--------------------------------------------------------------------------------
[2023-06-01T11:20:09.017+0000] {taskinstance.py:1284} INFO - Starting attempt 1 of 6
[2023-06-01T11:20:09.021+0000] {taskinstance.py:1285} INFO - 
--------------------------------------------------------------------------------
[2023-06-01T11:20:09.236+0000] {taskinstance.py:1304} INFO - Executing <Task(PostgresOperator): delete_data_from_table> on 2022-01-19 00:00:00+00:00
[2023-06-01T11:20:09.267+0000] {standard_task_runner.py:55} INFO - Started process 740 to run task
[2023-06-01T11:20:09.307+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'dag_with_postgres_operator_v03', 'delete_data_from_table', 'scheduled__2022-01-19T00:00:00+00:00', '--job-id', '133', '--raw', '--subdir', 'DAGS_FOLDER/postgres.py', '--cfg-path', '/tmp/tmpvst5h5gz']
[2023-06-01T11:20:09.321+0000] {standard_task_runner.py:83} INFO - Job 133: Subtask delete_data_from_table
[2023-06-01T11:20:09.773+0000] {task_command.py:389} INFO - Running <TaskInstance: dag_with_postgres_operator_v03.delete_data_from_table scheduled__2022-01-19T00:00:00+00:00 [running]> on host 687ba427e841
[2023-06-01T11:20:10.373+0000] {taskinstance.py:1513} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=dag_with_postgres_operator_v03
AIRFLOW_CTX_TASK_ID=delete_data_from_table
AIRFLOW_CTX_EXECUTION_DATE=2022-01-19T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-01-19T00:00:00+00:00
[2023-06-01T11:20:10.375+0000] {sql.py:253} INFO - Executing: 
            delete from dag_runs where dt = '2022-01-19' and dag_id = 'dag_with_postgres_operator_v03';
        
[2023-06-01T11:20:10.428+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2023-06-01T11:20:11.729+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2023-06-01T11:20:11.752+0000] {sql.py:364} INFO - Running statement: 
            delete from dag_runs where dt = '2022-01-19' and dag_id = 'dag_with_postgres_operator_v03';
        , parameters: None
[2023-06-01T11:20:11.759+0000] {sql.py:373} INFO - Rows affected: 0
[2023-06-01T11:20:11.779+0000] {taskinstance.py:1327} INFO - Marking task as SUCCESS. dag_id=dag_with_postgres_operator_v03, task_id=delete_data_from_table, execution_date=20220119T000000, start_date=20230601T112008, end_date=20230601T112011
[2023-06-01T11:20:11.955+0000] {local_task_job.py:159} INFO - Task exited with return code 0
[2023-06-01T11:20:12.026+0000] {taskinstance.py:2582} INFO - 1 downstream tasks scheduled from follow-on schedule check
